index.js

```diff
@@ -14,6 +14,8 @@ async function runAction() {
   const dockerfile = core.getInput('dockerfile') || '.';
   const registryHost = core.getInput('humanitec-registry') || 'registry.humanitec.io';
   const apiHost = core.getInput('humanitec-api') || 'api.humanitec.io';
+  const tag = core.getInput('tag') || null;
```

Comment:
`null` is normally not a good idea. Better to set it to something that will not blow up if it is used as the type you expect, e.g. empty string `''`
Note: `'' ? 'truthy' : 'falsey'` returns `'falsey`